hack/generator/pkg/astmodel/storage_conversion_factories.go

```diff
@@ -0,0 +1,360 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astmodel
+
+import (
+	"fmt"
+	"github.com/Azure/k8s-infra/hack/generator/pkg/astbuilder"
+	"github.com/dave/dst"
+	"github.com/pkg/errors"
+	"go/token"
+)
+
+// createPropertyConversion tries to create a property conversion between the two provided properties, using all of the
+// available conversion functions in priority order to do so.
+func createPropertyConversion(
+	sourceProperty *PropertyDefinition,
+	destinationProperty *PropertyDefinition,
+	knownLocals map[string]struct{}) (StoragePropertyConversion, error) {
```

Comment:
```suggestion
	knownLocals KnownLocalsSet) (StoragePropertyConversion, error) {
```

Reply:
No longer needed - I changed this into a method on `StorageConversionFunction`, which removes debris from the parameter list making the code more declarative.